test.js

```diff
@@ -25,3 +26,7 @@ test('min and max arguments set graph range', t => {
 	t.is(fn([10, 11, 12, 13], {min: 0}), '▇▇██');
 	t.is(fn([10, 20, 30, 40, 50], {max: 100}), '▁▂▃▄▄');
 });
+
+test('colored graph', t => {
+	t.is(fn([1, 2, 3, 4, 5, 6, 7, 8], {style: 'fire'}), colors.fg.getRgb(5, 5, 3) + '▁' + colors.fg.getRgb(5, 5, 4) + '▂' + colors.fg.getRgb(5, 5, 0) + '▃' + colors.fg.getRgb(5, 4, 0) + '▄' + colors.fg.getRgb(5, 3, 0) + '▅' + colors.fg.getRgb(5, 2, 0) + '▆' + colors.fg.getRgb(5, 1, 0) + '▇' + colors.fg.getRgb(5, 0, 0) + '█');
```

Comment:
No point in testing all the colors as you're just duplicating the main code. Testing the color of the first spark should be enough.